lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -46,7 +47,7 @@ internal sealed class TaskRuntime : IObserver<ICloseEvent>, IObserver<ISuspendEv
         [Inject]
         private TaskRuntime(
```

Comment:
This file has lots of (good) formatting changes mixed with a few actual code changes. Maybe revert the formatting changes to keep the change small?